src/emeus-vfl-parser.c

```diff
@@ -213,6 +213,22 @@ has_view (VflParser *parser,
   return g_hash_table_lookup (parser->views_set, name) != NULL;
 }
 
+/* Valid attributes */
+static const char *valid_attributes[] = {
```

Comment:
VFL is not going to be huge, but maybe we should start thinking about either interning these strings using `g_intern_static_string()` and then comparing pointers; or using something like `gperf` to generate perfect hashing, so we don't need to run a string comparison.

Not a blocker, but could be useful to investigate as a separate issue.